frontend/TestRun/ResultsTab.svelte

```diff
@@ -0,0 +1,113 @@
+<script>
+    import {onMount} from "svelte";
+
+    let fetching = true;
+    export let id = "";
+    export let test_id = "";
+    let results = [];
+
+
+    const fetchResults = async function () {
+        fetching = true;
+        try {
+            let apiResponse = await fetch(`/api/v1/run/${test_id}/${id}/fetch_results`);
+            let apiJson = await apiResponse.json();
+            if (apiJson.status === "ok") {
+
+                fetching = false;
+                results = apiJson.tables;
+                console.log(results);
+            }
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    const durationToStr = (seconds) => {
+        const hours = Math.floor(seconds / 3600);
+        const minutes = Math.floor((seconds % 3600) / 60);
+        const secs = seconds % 60;
+        return `${hours.toString().padStart(2, "0")}:${minutes.toString().padStart(2, "0")}:${secs.toString().padStart(2, '0')}`;
+    };
+
+    const formatValue = (value, type) => {
+        switch (type) {
+        case "FLOAT":
+            return value.toFixed(2);
+        case "INTEGER":
+            return value.toLocaleString();
+        case "DURATION":
+            return durationToStr(value);
+        default:
+            return value;
+        }
+    };
+
+    const getColumnType = (result, columnName) => {
+        return result.meta.columns_meta.find((col) => col.name === columnName).type;
+    };
+
+    onMount(() => {
+        fetchResults();
+    });
+
+    const styleMap = {
+        "PASS": "table-success",
+        "ERROR": "table-danger",
+        "WARNING": "table-warning"
+    };
+
+</script>
+
+<style>
+    th, td {
+        text-align: center;
+    }
+
+    table {
+        width: auto;
+        table-layout: auto;
+    }
+</style>
+
+{#if !fetching}
+    <div class="m-2">
```

Comment:
Looking at the screenshot, you may want to change this to `mb-2` or no margin at all, there's a grey line under the tab marker on yours.

Reply:
i needed also to be moved a bit from the left edge. Added.